Features/Context/SubContext/CommonActions.php

```diff
@@ -0,0 +1,363 @@
+    protected function waitForLastJs()
+    {
+        //Needs to be here
+        $this->execJavascript( '' );
+
+        $jsCode = "return BDD.isSomethingLoading();";
+
+        while ( $this->evalJavascript( $jsCode ) )
+        {
+            usleep( 100 * 1000 ); // 100ms
+        }
+    }
+
+    /**
+     * Waits for Javascript to finnish by running a empty Javascript
+     * (In Sahi it's possible to have the same result by running an empty javascript only)
+     */
+    protected function activateJsErrorHandler()
+    {
+        //Needs to be here
+       // $this->execJavascript( '' );
+        $jsCode = "return BDD.errorHandlerActivate();";
+        $this->execJavascript( $jsCode );
+    }
+
+    /**
+     * Evaluate javascript code and return result.
+     *
+     * @param  string  $jsCode
+     * @return mixed
+     */
+    protected function evalJavascript( $jsCode, $addJsHelper = true )
+    {
+        if ( $addJsHelper )
+        {
+            $jsCode = JsHelper::getHelperJs() . $jsCode;
+        }
+        $jsCode = $this->driver->wrappedFunction( $jsCode );
+
+        return $this->getSession()->evaluateScript( $jsCode );
+    }
+
+    /**
+     * Execute javascript code and return result.
+     *
+     * @param  string  $jsCode
+     * @return mixed
+     */
+    protected function execJavascript( $jsCode, $addJsHelper = true )
+    {
+        if ( $addJsHelper )
+        {
+            $jsCode = JsHelper::getHelperJs() . $jsCode;
+        }
+
+        $jsCode = $this->driver->wrappedFunction( $jsCode );
+
+        return $this->getSession()->executeScript( $jsCode );
+    }
+
+    /**
+     * Clicks on the PlatformUI logo
+     *
+     * @param string $text
+     */
+    protected function clickLogo()
+    {
+        $selector = 'img[alt="Logo eZ"]';
+        $jsArg = JsHelper::generateFuncArgs( $selector );
+        $jsCode = "return BDD.clickElement( $jsArg );";
+        $this->execJavascript( $jsCode );
+    }
+
+    /**
+     * Clicks on a PlatformUI tab
+     *
+     * @param string $text
+     */
+    protected function clickTab( $text )
+    {
+        $this->clickElementsByText( $text, ".ez-tabs-label a[href]" );
+    }
+
+    /**
+     * Click on a PlatformUI menu zone
+     *
+     * @param string $text
+     */
+    protected function clickMenuZone( $text )
+    {
+        $this->clickElementsByText( $text, ".ez-zone-name" );
+    }
+
+    /**
+     * Over on a PlatformUI menu zone
+     *
+     * @param string $text
+     */
+    protected function overMenuZone( $text )
+    {
+        $selector = ".ez-zone-name";
+        $jsArgs = JsHelper::generateFuncArgs( $text, $selector );
+        $jsCode = "return BDD.mouseOverElementByText( $jsArgs );";
+        $this->execJavascript( $jsCode );
+    }
+
+    /**
+     * Click on a PlatformUI menu link
+     *
+     * @param string $text
+     */
+    protected function clickMenuLink( $text )
+    {
+        $this->clickElementsByText( $text, '.ez-link a[href]' );
```

Comment:
the `ez-link` links is gone for quite some time now, so I guess this method can be removed